### PR TITLE
feat: add Google Slides and Google Forms

### LINKED ISSUE #15: [Bug] Unable to switch account on Google Slides
Oddly, Account Switcher doesn't work on the following page:
https://docs.google.com/presentation/u/0/

### LINKED ISSUE #16: [Bug] Unable to switch account on Google Forms
Account Switcher doesn't work on the following page:
https://docs.google.com/forms/u/0/

### PR DESCRIPTION
**Pull request by jaeyongjaykim**
This PR closes #15 and closes #16.

Applied formatting and added to handle two services (Slides and Forms)

### CODE REVIEW

**Comment by jaeyongjaykim** [comment]
@anshulahuja98 Would you plase be able to merge this PR and release new version?

**Comment by anshulahuja98** [comment]
I had left a pr feedback, can you please address that

**Comment by anshulahuja98** [comment]
@jaeyongjaykim  pl address PR feedback so that it can be checked in

**Comment by jaeyongjaykim** [comment]
> pl address PR feedback so that it can be checked in

I don't see any feedback on this PR. Where would I find that? @anshulahuja98

**Review by anshulahuja98 — CHANGES_REQUESTED** [review]

**Comment by anshulahuja98** [comment]
Seems like I forgot to click submit after the comment
My bad
Can you check now, a comment should show up.

@jaeyongjaykim

File: background.js [hunk 1]
```diff
@@ -10,42 +9,49 @@ chrome.runtime.onMessage.addListener(function (command) {
...
-    if (current_url.match(drive_regex)) {
+    // Google Slides
+    const slides_regex = "https:\/\/docs.google.com\/presentation\/u\/[0-9].*";
+    if (current_url.match(slides_regex)) {
       update_url_regex = RegExp("u\/[0-9]");
-      update_acc = "u/" + account_num
+      update_acc = "u/" + account_num;
     }
 
-    // Google Play Music
```
[the remark this answers is not in the document]
Comment (jaeyongjaykim) [reply background.js:37]: image

> can you please add play music back to this list.
Seems that it got removed.

The reason I've removed Google Play Music is it's no longer serviced, and replaced by YouTube Music. @anshulahuja98

**Comment by jaeyongjaykim** [comment]
> Can you check now, a comment should show up.

I've commented on your review. @anshulahuja98

File: background.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (anshulahuja98) [reply background.js:37]: ah good catch

**Review by anshulahuja98 — APPROVED** [review]

**Comment by anshulahuja98** [comment]
Thank you for your contribution @jaeyongjaykim

**Comment by jaeyongjaykim** [comment]
> Thank you for your contribution

Thank YOU for letting me contribute your beautiful work! @anshulahuja98

**Pull request merged**